services/tools/wanaku-routing-yaml-route-service/src/main/resources/application.properties

```diff
@@ -23,4 +23,7 @@ wanaku.service.routing.base-uri=direct:start
 # valkey.port=6379
 
 # Timeout
-# valkey.timeout=10
\ No newline at end of file
+# valkey.timeout=10
+
+#quarkus.mongodb.connection-string = mongodb://localhost:27017
+#wanaku.persistence.mongodb = true
```

Comment:
I'm a bit confused here. The downstream services should not have access to the DB. The only medium that they use is the service registry (Valkey).

Reply:
This is not the case anymore, I'll remove these, only the wanaku-server-quarkus itself need the persistence layer, the services only need K/V store